Change order of includes to fix compilation with musl libc

--- a/include/headers.h	2021-08-16 14:27:51.000000000 -0300
+++ b/include/headers.h	2021-10-01 23:42:09.471528972 -0300
@@ -106,11 +106,6 @@
 #include <net/if.h>
 #endif
 
-#if ((HAVE_TUNTAP_TAP) || (HAVE_TUNTAP_TUN))
-#include <linux/if_tun.h>
-#include <sys/ioctl.h>
-#endif
-
 
 // AF_PACKET HEADERS
 #if defined(HAVE_LINUX_FILTER_H) && defined(HAVE_AF_PACKET)
@@ -146,6 +141,11 @@
 #define  IPV6HDRLEN 40
 #endif // HAVE_AF_PACKET
 
+#if ((HAVE_TUNTAP_TAP) || (HAVE_TUNTAP_TUN))
+#include <linux/if_tun.h>
+#include <sys/ioctl.h>
+#endif
+
 #ifdef WIN32
 
 /* Windows config file */
